packages/metro-config/index.js

```diff
@@ -36,6 +39,53 @@ const INTERNAL_CALLSITES_REGEX = new RegExp(
   ].join('|'),
 );
 
+/**
+ * Resolves the root of an NPM or Yarn workspace, by traversing the file tree upwards from a `candidatePath` in the search for
+ * - a directory with a package.json
+ * - which has a `workspaces` array of strings
+ * - which (possibly via a glob) includes the project root
+ * @param {string} projectRoot Project root to find a workspace root for
+ * @param {string | undefined} candidatePath Current path to search from
```

Comment:
Nit: As we have parameter types defined in code (Flow), it's redundant to repeat them in JSDoc (although we don't currently perform any processing on this — but let's simplify maintenance).

```suggestion
 * @param projectRoot Project root to find a workspace root for
 * @param candidatePath Current path to search from
```

IMO we could **omit `@param/@returns` entirely**, since `projectRoot` is self-documenting and `candidatePath` is explained in the first line.

Reply:
I've removed the `@param` and `@returns` 👍